gax/src/test/java/com/google/api/gax/core/DistributionTest.java

```diff
@@ -29,66 +29,44 @@
  */
 package com.google.api.gax.core;
 
-import com.google.common.truth.Truth;
-import org.junit.Before;
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.Arrays;
+import java.util.Collections;
 import org.junit.Test;
 
 public class DistributionTest {
-  private Distribution distribution;
-
-  @Before
-  public void setup() {
-    distribution = new Distribution(10);
-  }
-
+  // These tests come from examples in https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method
   @Test
-  public void testGetCount() {
-    distribution.record(0);
-    distribution.record(1);
-    distribution.record(100);
-    Truth.assertThat(distribution.getCount()).isEqualTo(3);
-  }
+  public void testPercentile() {
```

Comment:
You no longer have a test for a bucket value that is higher than `totalBuckets`.

Reply:
Done